lib/edit/style/SLDStyleConverter.js

```diff
@@ -98,6 +101,10 @@ exports.SLDStyleConverter = function() {
                         wellKnownName: style.symbol && style.symbol.shape || 'circle'
                     }];
             }
+            var opacity = 1;
+            if (style.symbol && angular.isDefined(style.symbol.fillOpacity)) {
+                opacity = Math.max(0.01, style.symbol.fillOpacity) / 100;
```

Comment:
hey @ischneider one thing I am wondering here (but I don't know the SLD spec specifics nor the GeoServer internals), will this not lead to the fill opacity being "executed" twice? i.e. 80*80% = 64% ?